api/v1/controllers/aspects.js

```diff
@@ -187,6 +191,34 @@ module.exports = {
   },
 
   /**
+   * DELETE /aspects/{keys}/writers
+   *
+   * Deletes all the writers associated with this resource.
+   *
+   * @param {IncomingMessage} req - The request object
+   * @param {ServerResponse} res - The response object
+   * @param {Function} next - The next middleware function in the stack
+   */
+  deleteAspectWriters(req, res, next) {
+    doDeleteAllAssoc(req, res, next, helper, helper.belongsToManyAssoc.users);
+  },
+
+  /**
+   * DELETE /aspects/{keys}/writers/userNameOrId
+   *
+   * Deletes a user from an aspect’s list of authorized writers.
+   *
+   * @param {IncomingMessage} req - The request object
+   * @param {ServerResponse} res - The response object
+   * @param {Function} next - The next middleware function in the stack
+   */
+  deleteAspectWriter(req, res, next) {
+    const userNameOrId = req.swagger.params.userNameOrId.value;
+    doDeleteOneAssoc(req, res, next, helper,
+        helper.belongsToManyAssoc.users, userNameOrId);
```

Comment:
@shriramshankar is there a reason why we need to pass both helper and helper.belongsToManyAssoc.users?